server/handlers/converters.go

```diff
@@ -59,7 +59,10 @@ func (ah *ActivityHandler) msgToPost(channelID, senderID string, msg *msteams.Me
 		text = "## " + msg.Subject + "\n" + text
 	}
 
-	post := &model.Post{UserId: senderID, ChannelId: channelID, Message: text, Props: props, RootId: rootID, FileIds: attachments}
+	post := &model.Post{UserId: senderID, ChannelId: channelID, Message: text, Props: props, RootId: rootID}
+	if isUpdatedActivity {
+		post.FileIds = attachments
+	}
```

Comment:
shouldn't we add file IDs in all cases?

Reply:
It is a PATCH API. We should only send those fields which we want to update.